src/context/SettingsContext/SettingsContext.provider.tsx

```diff
@@ -32,6 +34,32 @@ export const SettingsProvider: React.FC<{ children: ReactNode }> = ({
     setCookie('settings', settings, { maxAge: 2592000 });
   }, [settings]);
 
+  useEffect(() => {
+    const handleFontSizeGuard = () => {
+      const screenWidth = window.innerWidth;
+      if (settings.fontSize > 1) {
+        if (screenWidth <= breakpoints.laptop && settings.fontSize > 1.4) {
```

Comment:
zrób to `switch`

Reply:
zrobiłem to i if-else z Toolboxa (powiązany z tym).